### PR TITLE
Add a configuration option for simple logging.

### PR DESCRIPTION
**Pull request by armstnp**

### CODE REVIEW

File: lib/utils/requestLogger.js [hunk 1]
```diff
@@ -55,7 +55,7 @@ exports.create = function(logger) {
 
     var preview = JSON.stringify(censoredPayload);
     if(preview.length > PAYLOAD_PREVIEW_LENGTH) {
-      preview = payload.substring(0, PAYLOAD_PREVIEW_LENGTH) + '...';
+      preview = preview.substring(0, PAYLOAD_PREVIEW_LENGTH) + '...';
```
Comment (cameronbowie) [inline lib/utils/requestLogger.js:58]: Was this a bug?

File: index.js [hunk 2]
```diff
@@ -20,9 +20,48 @@ function buildRequestor(clientOptions) {
 };
 
 function buildLogger(clientOptions) {
-  return clientOptions.logger
-    || (clientOptions.logContainer && clientOptions.logContainer.get('smartsheet'))
-    || winston.loggers.get('smartsheet');
+  if((clientOptions.logger && clientOptions.loggerContainer)
+      || (clientOptions.logger && clientOptions.logLevel)
+      || (clientOptions.loggerContainer && clientOptions.logLevel)) {
```
Comment (cameronbowie) [inline index.js]: Can you extract this boolean logic into a helper method?

File: README.md [hunk 3]
```diff
@@ -85,13 +85,19 @@ When creating the client object, pass an object with any of the following option
 
 This library leverages [**winston**](https://github.com/winstonjs/winston) for logging.  **winston** provides [documentation on configuring loggers](https://github.com/winstonjs/winston#working-with-multiple-loggers-in-winston).  To receive logs from the SDK, choose one of the following:
 
-* Create and configure a **winston** container named 'smartsheet'. _(Recommended)_
-* Create and configure a **winston** logger named 'smartsheet'.
+### Simple
 
-If you choose not to use **winston** and want to use your own logger, pass a logger object in the client configuration under the key `logger` that meets the following specification:
+Pass the `logLevel` configuration option as one of the default **winston** log levels: `silly`, `verbose`, `debug`, `info`, `warn`, or `error`.  This will default to a simple Console transport.
```
Comment (cameronbowie) [inline README.md:90]: Are we adding code samples or is Kevin? It seems like each of these sections should have an example

File: lib/utils/requestLogger.js (hunk 1 above)
Comment (armstnp) [reply lib/utils/requestLogger.js:58]: Thanks for rubbing it in. 😛

File: index.js (hunk 2 above)
Comment (armstnp) [reply index.js]: 👍

File: README.md (hunk 3 above)
Comment (armstnp) [reply README.md:90]: Can add examples afterward, but this is a good plan.

**Pull request merged**